### PR TITLE
Release 0.2.0 Updates

### PR DESCRIPTION
**Pull request by simpsont-oci**
Updates in preparation for release 0.2.0 of node-opendds

To be merged after #50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,24 +5,25 @@
...
 ```
-$ # In OpenDDS directory.
 $ source setenv.sh
-$ # In node-opendds directory
+```
+* In node-opendds directory:
+```
 $ npm install
-$ mwc.pl -type gnuace
+$ mwc.pl -type gnuace .
```
Comment (mitza-oci) [inline README.md:26]: Does this make a difference?  It doesn't seem like it would.

File: package.json [hunk 2]
```diff
@@ -18,10 +18,15 @@
     "opendds",
     "addon"
   ],
-  "author": "Adam Mitz (Object Computing, Inc.)",
+  "contributors": [
+    "Adam Mitz <mitza@objectcomputing.com> (https://github.com/mitza-oci)",
+    "Fred Hornsety <hornseyf@objectcomputing.com> (https://github.com/iguessthislldo)",
```
Comment (mitza-oci) [inline package.json:23]:
```suggestion
    "Fred Hornsey <hornseyf@objectcomputing.com> (https://github.com/iguessthislldo)",
```

File: README.md [hunk 3]
```diff
@@ -5,24 +5,25 @@
 * Make sure all of the OpenDDS development environment variables are set before running npm to install this module.  If you use the OpenDDS configure script, these variables can be set using "setenv.sh" or "setenv.cmd" depending on your platform.
 ** The OpenDDS development environment can use either the source tree or the installed tree (the one created by "make install").
 ** If using the installed tree, set the DDS_ROOT environment variable using the installed dds-devel.sh script.
+** If using security, you may need to ensure that OpenDDS is built with the same version of openssl used by Node.js
```
Comment (mitza-oci) [inline README.md:8]:
```suggestion
** If using security, you may need to ensure that OpenDDS is built with the same version of OpenSSL used by Node.js
```

**Review by mitza-oci — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (simpsont-oci) [reply README.md:26]: Not really, it just seems clearer to me.

Comment (mitza-oci) [reply README.md:26]: It's like `find` (at least the GNU version).  If you don't specify anything else it works on the current directory.

**Review by mitza-oci — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -35,6 +36,14 @@ $ node-gyp configure build
 
 ## Changelog
 
+### Version 0.2
```
Comment (iguessthislldo) [inline README.md:39]:
```suggestion
### Version 0.2.0
```

To be consistent with 0.1.0 below and OpenDDS

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "opendds",
-  "version": "0.1.1",
+  "version": "0.2",
```
Comment (iguessthislldo) [inline package.json:3]:
```suggestion
  "version": "0.2.0",
```

**Review by iguessthislldo — CHANGES_REQUESTED** [review]

**Review by iguessthislldo — APPROVED** [review]

**Pull request merged**